nix-ts-mode.el

```diff
@@ -172,14 +172,16 @@
      ((node-is "]") parent-bol 0)
      ((node-is ")") parent-bol 0)
      ((node-is "}") parent-bol 0)
+     ((node-is "''") parent-bol 0)
      ((node-is "binding_set") parent-bol nix-ts-mode-indent-offset)
      ((node-is "indented_string_expression") parent-bol nix-ts-mode-indent-offset)
      ((parent-is "formals") parent-bol 0)
      ((parent-is "binding_set") parent-bol 0)
      ((parent-is "binding") parent-bol nix-ts-mode-indent-offset)
      ((parent-is "list_expression") parent-bol nix-ts-mode-indent-offset)
      ((parent-is "apply_expression") parent-bol nix-ts-mode-indent-offset)
-     ((parent-is "parenthesized_expression") parent-bol nix-ts-mode-indent-offset)))
+     ((parent-is "parenthesized_expression") parent-bol nix-ts-mode-indent-offset)
+     (no-node parent-bol nix-ts-mode-indent-offset))) ; Automatically indent as soon as a new (empty) line is created
```

Comment:
In [your screencast at mark 0:40](https://asciinema.org/a/Tbb6oDw3uutGTDMPFUz1JJpNo?t=40), it looks like the second line in the indented string is  indented one level further than it ought to be.

I believe this might have to be 0, to follow the same indent level as the previous line:
```suggestion
     (no-node parent-bol 0))) ; Automatically indent as soon as a new (empty) line is created
```

Reply:
For this reason I wrote that it doesn't work well :)

The problem is that if you don't close the string with another `''` it's like the following lines in the string were like arguments of the first one, so the behavior makes sense. Indeed if you close the string with another `''` and then press `RET` on those lines they are indented correctly. The reality is that I've no idea what to do when the tree contains `ERROR` nodes.

Moreover your suggested change is not correct because in Nix multi lines strings for convention are always indented. Or perhaps we should make it strings at the same level of the parent, wait the the user closes the multi lines string and then let him re-indent the string as he wants.

Anyway to do that I took inspiration by what `typescript-ts-node` does which to be honest I don't think follows the typescript convention (but I'm not sure about this). I should try checking what other tree sitter modes for other languages do.